### PR TITLE
[Jetnews] Update to Material3

### PR DESCRIPTION
**Pull request by JolandaVerhoef**

### CODE REVIEW

File: JetNews/app/build.gradle.kts [hunk 1]
```diff
@@ -97,17 +97,30 @@ dependencies {
...
+    debugImplementation("androidx.compose.ui:ui-tooling:1.3.0-beta01")
+//    implementation(libs.androidx.compose.animation)
+//    implementation(libs.androidx.compose.foundation.layout)
+//    implementation(libs.androidx.compose.material.iconsExtended)
+//    implementation(libs.androidx.compose.material3)
+//    implementation(libs.androidx.compose.materialWindow)
+//    implementation(libs.androidx.compose.runtime.livedata)
+//    implementation(libs.androidx.compose.ui.tooling.preview)
+//    debugImplementation(libs.androidx.compose.ui.test.manifest)
+//    debugImplementation(libs.androidx.compose.ui.tooling)
```
Comment (arriolac) [inline JetNews/app/build.gradle.kts:118]: Remove? Do you also want to add the dependencies to the .toml file or is that what the TODO is for?

File: JetNews/app/src/main/java/com/example/jetnews/ui/home/HomeRoute.kt [hunk 2]
```diff
@@ -78,9 +77,8 @@ fun HomeRoute(
  * @param onInteractWithArticleDetails (event) indicate that the article details were interacted
  * with
  * @param openDrawer (event) request opening the app drawer
- * @param scaffoldState (state) state for the [Scaffold] component on this screen
+// * @param snackbarHostState (state) state for the [Scaffold] component on this screen
```
Comment (arriolac) [inline JetNews/app/src/main/java/com/example/jetnews/ui/home/HomeRoute.kt:80]: Remove '//'?

File: JetNews/app/src/main/java/com/example/jetnews/ui/interests/InterestsRoute.kt [hunk 3]
```diff
@@ -28,14 +28,14 @@ import androidx.compose.runtime.saveable.rememberSaveable
...
- * @param scaffoldState (state) state for screen Scaffold
+ * @param snackbarHostState (state) state for screen snackbar host
  */
 @Composable
 fun InterestsRoute(
     interestsViewModel: InterestsViewModel,
     isExpandedScreen: Boolean,
     openDrawer: () -> Unit,
-    scaffoldState: ScaffoldState = rememberScaffoldState()
+    snackbarHostState: SnackbarHostState = remember { SnackbarHostState() }
```
Comment (arriolac) [inline JetNews/app/src/main/java/com/example/jetnews/ui/interests/InterestsRoute.kt:38]: Should we have a `rememberSnackbarHostState` method defined? I see this pattern used elsewhere

**Review by arriolac — APPROVED** [review]
LGTM just some minor comments

File: JetNews/app/build.gradle.kts (hunk 1 above)
Comment (JolandaVerhoef) [reply JetNews/app/build.gradle.kts:118]: The todo is to remove the explicit dependencies and use these commented out versions again, once all Compose & Material3 dependencies align again

File: JetNews/app/src/main/java/com/example/jetnews/ui/interests/InterestsRoute.kt (hunk 3 above)
Comment (JolandaVerhoef) [reply JetNews/app/src/main/java/com/example/jetnews/ui/interests/InterestsRoute.kt:38]: I don't know? It's not part of the public API, so I just used what's being used elsewhere. Something to discuss with @astamato maybe?

Comment (astamato) [reply JetNews/app/src/main/java/com/example/jetnews/ui/interests/InterestsRoute.kt:38]: Interesting, I'd leave as is for now. All the rememberX functions ive come across do smth more complex than just remember a single value.

**Pull request merged**